platform/branding-api/pom.xml

```diff
@@ -21,9 +21,8 @@
         <artifactId>platform</artifactId>
         <version>2.11.0-SNAPSHOT</version>
     </parent>
-    <groupId>ddf.distribution</groupId>
     <artifactId>branding-api</artifactId>
```

Comment:
This `artifactId` could be `platform-branding-api` instead to match the other modules in ddf/platform.